src/mr-importer/usd_loader.cpp

```
@@ -0,0 +1,815 @@
+  size_t const nv = scratch.positions.size();
+  for (int vi : scratch.face_vertex_indices) {
+    if (vi < 0 || static_cast<size_t>(vi) >= nv) {
+      return false;
+    }
+  }
+
+  scratch.normals_xyz.clear();
+  scratch.uv_xy.clear();
+
+  if (is_enabled(options, Options::LoadMeshAttributes)) {
+    UsdGeomPrimvarsAPI pvapi(geom.GetPrim());
```

Clang (14.0.6)

[warning] variable 'pvapi' is not initialized

(cppcoreguidelines-init-variables)